src/frontend/admin/src/components/templates/certificates-definitions/form/CertificateDefinitionForm.tsx

```diff
@@ -64,7 +67,7 @@ export function CertificateDefinitionForm({ definition, ...props }: Props) {
       name: definition?.name ?? "",
       title: definition?.title ?? "",
       description: definition?.description ?? "",
-      template: "howard.issuers.CertificateDocument",
+      template: "certificate",
```

Comment:
This diff is not in the scope of this PR. At least move this diff in a dedicated commit.

Furthermore, I'll use an enum to declare certificate template types